datasets/funderingen/pandFunderingen/v0.0.1.json

```diff
@@ -0,0 +1,88 @@
+  "authorizationGrantor": "datateamSOB@amsterdam.nl",
+  "owner": "Gemeente Amsterdam",
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "identifier": "pandIdentificatie",
+    "required": [
+      "pandIdentificatie",
+      "schema"
+    ],
+    "display": "pandIdentificatie",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "pandIdentificatie": {
+            "type": "string",
+            "description": "Unieke aanduiding van het pand."
+          },
+          "bouweenheid": {
+            "type": "string",
+            "description": "Refereert aan de bouweenheid waarin het pand zich bevindt, relevant bij het analyseren van funderingsinformatie over grotere bouwprojecten of complexen. Een bouweenheid kan panden omvatten met verschillende straatnamen en huisnummers."
+          },
+          "bouwvergunningJaar": {
+            "type": "string",
+            "description": "Het jaar/periode waarop de bouwvergunning is uitgegeven."
+          },
+          "bestemmingGebouw": {
+            "type": "string",
+            "description": "De functie van het gebouw (bijvoorbeeld woonfunctie, commerciële functie), relevant voor de waarde van het gebouw en de urgentie van funderingsherstel."
+          },
+          "opmerkingen": {
+            "type": "string",
+            "description": "Aanvullende informatie, zoals de hoogte van de tuin ten opzichte van de straat en specifieke funderingsrisico’s."
+          },
+          "typeFundering": {
+            "type": "string",
+            "description": "Het type fundering met specifieke aanduidingen."
+          },
+          "bovenkant": {
+            "type": "string",
+            "description": "De hoogte van de fundering ten opzichte van het maaiveld of NAP."
+          },
+          "aanlegtype": {
+            "type": "string",
+            "description": "Het type aanleg van de fundering. Dit kan bijvoorbeeld relevant zijn voor de evaluatie van de oorspronkelijke bouw- en renovatietechnieken."
+          },
+          "lengteOplanger": {
+            "type": "string",
+            "description": "Lengtes van funderingselementen zoals palen of oplangers."
+          },
+          "bovenkantPalen": {
+            "type": "string",
+            "description": "Lengtes van funderingselementen zoals palen of oplangers."
+          },
+          "kwaKlasse": {
+            "type": "string",
+            "description": "Geeft de kwaliteitsklasse van de fundering aan."
+          },
+          "hoogteMaaiveld": {
+            "type": "string",
+            "description": "De hoogte van het maaiveld ten opzichte van NAP, wat belangrijk is bij het beoordelen van de invloed van grondwaterstanden en verzakkingen."
+          },
+          "souTerrain": {
```

Comment:
Is er een reden dat dit veld straks als `sou_terrain` in de database moet komen? Lijkt me een typefoutje?